po/de.po

```diff
@@ -5885,8 +5885,8 @@ msgid ""
 "Exchange by CSV/FinTS(former HBCI) or import SWIFT-MT9xx/QIF/OFX data "
 "including Transaction Matching"
 msgstr ""
-"Datenaustausch mittels CSV/FinTS(former HBCI) oder importiere SWIFT-MT9xx/"
-"QIF/OFX Daten, einschließlich Buchungsabgleich"
+"Datenaustausch mittels CSV/FinTS oder importiere SWIFT-MT9xx/QIF/OFX Daten, "
+"einschließlich Buchungsabgleich"
```

Comment:
Christian, schreib doch bitte in solchen Fällen Kommentare gegen den Quelltext rein. Dann können wir die Liste Peu a Peu abarbeiten.